kinto/plugins/accounts/__init__.py

```diff
@@ -29,6 +41,20 @@ def includeme(config):
 
```

Comment:
Please add conditional view loading for reset password and validation.

Reply:
It's done now ;)